database/migrations/2022_11_27_143608_bump_version040603.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
-use App\Models\Configs;
 use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
 
 class BumpVersion040603 extends Migration
```

Comment:
The new migration created by Laravel are all named with `return new class() extends Migration`

I would avoid renaming old migrations as this may trigger a re-run.

Reply:
I just tested and can confirm your statement.